DataManagementSystem/Client/test/Test_Client_DataManagementSystem.py

```diff
@@ -41,7 +41,7 @@ class ConsistencyInspectorSuccess(UtilitiesTestCase):
 
   def test_getReplicasPresence(self):
     res = self.ci.getReplicasPresence(['/this/is/dir1/file1.txt', '/this/is/dir1/file2.foo.bar'])
-    self.assertEqual(res, (['/this/is/dir1/file1.txt', '/this/is/dir1/file2.foo.bar'], []))
+    self.assertEqual(tuple(map(set, res)), ({'/this/is/dir1/file1.txt', '/this/is/dir1/file2.foo.bar'}, set()))
```

Comment:
why changing that ?

Reply:
`getReplicasPresence` uses dictionaries internally and doesn't maintain the order consistently. The fact that is worked in Python 2 is pure luck.

Reply:
Maybe just add a comment, cause it makes it look very obscure :-)
You say it's pure luck in Python 2, but Python 3 is still random when it comes to dictionary order, isn't it ?

Reply:
Python 3.7 guarantees insertion order: https://stackoverflow.com/a/39980744

Reply:
Ah forge the question, I just found in https://docs.python.org/3/tutorial/datastructures.html

> Performing list(d) on a dictionary returns a list of all the keys used in the dictionary, in insertion order (if you want it sorted, just use sorted(d) instead). To check whether a single key is in the dictionary, use the in keyword.